src/components/sidebar-navigation/SidebarNavigationItemStyles.tsx

```diff
@@ -54,8 +56,49 @@ export const sidebarNavigationBaseItemStyles = css`
 `;
 
 export const SidebarNavigationItemLabel = styled.span`
-  flex: 1 0 auto;
+  flex: 1 1 auto;
 
   ${truncate}
 `;
 SidebarNavigationItemLabel.displayName = 'SidebarNavigationItemLabel';
+
+export const itemIconStyles = css`
```

Comment:
Should we match the naming of the other exported styles from this file? Something like `sidebarNavigationItemIconStyles`?